src/tracks.ts

```diff
@@ -0,0 +1,39 @@
+import { Point } from "./points";
+
+// Definition of a specific track
+export interface Track {
+    dim: Point,
+    grid: number;
+    startLine: [Point, Point];
+    finishLine: [Point, Point];
+
+    trackWidth: number;
+    path: Point[];
+}
+
+export const U_TRACK:Track = {
+    dim: [400, 400],
+    grid: 20,
+    startLine: [[20, 10], [20, 110]],
+    finishLine: [[20, 290], [20, 390]],
+    trackWidth: 100,
+    path: [[20, 60], [340, 60], [340, 340], [20, 340]],
+}
+
+export const OVAL:Track = {
+    dim: [800, 400],
+    grid: 20,
+    startLine: [[400, 10], [400, 110]],
+    finishLine: [[380, 10], [380, 110]],
+    trackWidth: 100,
+    path: [[400, 60], [740, 60], [740, 340], [60, 340], [60, 60], [380, 60]],
+}
+
+export const BIG_OVAL:Track = {
+    dim: [800, 400],
+    grid: 10,
```

Comment:
This is similar to OVAL but has a grid size of 10 so it ends up being a longer course.